GG/src/Menu.cpp

```diff
@@ -519,7 +521,13 @@ void PopupMenu::Render()
                 glColor3ub(clr.r, clr.g, clr.b);
 
                 if (!menu.next_level[j].separator) {
-                    m_font->RenderText(line_rect.ul, line_rect.lr, menu.next_level[j].label, fmt);
+                    // TODO cache line data v expensive calculation
+                    std::vector<boost::shared_ptr<Font::TextElement> > text_elements
+                        = m_font->ExpensiveParseFromTextToTextElements(menu.next_level[j].label, fmt);
```

Comment:
I've moved the `=` operator.

I usually place the operator at the beginning of the line so that neither line is a complete line of code. Then it is clear from looking at the first character of continuing lines that they are continuing.   

I know that it not how it is done around here for `=` operators, but it is a default behavior.